### PR TITLE
Fixes a bunch of helper scripts for improved usability

### PR DESCRIPTION
**Pull request by mmattel**
This PR fixes many necessary helper scripts which eases maintenance and different repo setups.

* find orphand adoc files
-- fix missed hits,
single multi repo capability (one script for all repos!)
* find orphand attachments/example/images
-- create a uniformed master script which is called by a task script defining the environment,
-- single and multi repo capability (one script for all repos!)
* rename the `manual_config` script to `manual_config_pdf`
-- ease of differenciation
* fixed an image filename which contained a blank (filenames with blanks break the find logic)

Backport to 2.10, 2.9 and 2.8 (must be done immediately post merging)

### CODE REVIEW

File: bin/find-orphaned-adoc-files [hunk 1]
```diff
@@ -2,213 +2,280 @@
...
+	#exit
+
+	# get all files referenced via xref in the navigation file
+	nav_list=(`get_antora_nav_list "$NAV_FILE"`)
 	#echo "${nav_list[@]}" | tr ' ' '\n'
+	#exit
 
-	# create an array diff of uniqe files not in nav w/o partials as they are never in the nav
-	not_in_nav=($(echo ${not_partial_files[@]} ${nav_list[@]} | tr ' ' '\n' | sort | uniq -u))
+	# create an array diff of uniqe files not referenced in the navigation
```
Comment (phil-davis) [inline bin/find-orphaned-adoc-files:155]:
```suggestion
	# create an array diff of unique files not referenced in the navigation
```

File: bin/find-orphaned-attachment-files [hunk 2]
```diff
@@ -1,33 +1,27 @@
...
+TARGET_DIR="assets/attachments"
+# depending on the target, there may be a trailing slash necessary or not
+# check how the file is included
+SEARCH_PATTERN='link:{attachmentsdir}/'
 
-echo
-echo "Checking for orphaned ATTACHMENT files in manual: ${AVAILABLE_MANUAL}"
-echo "Please run this scrip from your docs root"
-echo
+# get the full path the the directory the script runs
```
Comment (phil-davis) [inline bin/find-orphaned-attachment-files:20]:
```suggestion
# get the full path of the directory where the script runs
```

File: bin/find-orphaned-base [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+			found_grep+=($(grep -rEow "${grep_list}" "${BASE_DIR}/pages" | sed "s|.*${SEARCH_PATTERN}:||g" | sed -e 's/^\///' | sort | uniq))
+			#echo "${found_grep[@]}" | tr ' ' '\n'
+			#exit
+		fi
+
+		# create an array diff and print to output
+		not_linked=($(echo ${found_find[@]} ${found_grep[@]} | tr ' ' '\n' | sort | uniq -u))
+		echo ${not_linked[@]} | tr ' ' '\n'
+
+		# count the number of occurences
```
Comment (phil-davis) [inline bin/find-orphaned-base:98]:
```suggestion
		# count the number of occurrences
```

File: bin/find-orphaned-base [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+		unset found_grep
+
+		# the base directory name depends if there is a single or are multiple manuals available
+		if [[ ${manual} = ${ROOT_NAME} ]]; then
+			BASE_DIR="./modules/${manual}"
+		else
+			BASE_DIR="./modules/${manual}${MANUAL_POSTFIX}"
+		fi
+
+		#echo "${BASE_DIR}"
```
Comment (phil-davis) [inline bin/find-orphaned-base:48]: To you want to leave these commented-out debugging lines here?

File: bin/find-orphaned-example-files [hunk 5]
```diff
@@ -1,33 +1,27 @@
...
-echo
-echo "Checking for orphaned EXAMPLE files in manual: ${AVAILABLE_MANUAL}"
-echo "Please run this scrip from your docs root"
-echo
+# get the full path the the directory the script runs
+DIR="$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"
 
-for manual in "${AVAILABLE_MANUAL[@]}"; do
-    [[ ! -e "./modules/${manual}/" ]] && echo "Cannot find ${manual} manual." && continue
+# load the workinng script
```
Comment (phil-davis) [inline bin/find-orphaned-example-files:23]:
```suggestion
# load the working script
```

File: bin/find-orphaned-attachment-files [hunk 6]
```diff
@@ -1,33 +1,27 @@
...
+DIR="$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"
 
-for manual in "${AVAILABLE_MANUAL[@]}"; do
-    [[ ! -e "./modules/${manual}/" ]] && echo "Cannot find ${manual} manual." && continue
+# load the workinng script
```
Comment (phil-davis) [inline bin/find-orphaned-attachment-files:23]:
```suggestion
# load the working script
```

File: bin/find-orphaned-image-files [hunk 7]
```diff
@@ -1,33 +1,26 @@
...
-echo
-echo "Checking for orphaned IMAGE files in manual: ${AVAILABLE_MANUAL}"
-echo "Please run this scrip from your docs root"
-echo
+# get the full path the the directory the script runs
+DIR="$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"
 
-for manual in "${AVAILABLE_MANUAL[@]}"; do
-    [[ ! -e "./modules/${manual}/" ]] && echo "Cannot find ${manual} manual." && continue
+# load the workinng script
```
Comment (phil-davis) [inline bin/find-orphaned-image-files:22]:
```suggestion
# load the working script
```

File: bin/find-orphaned-example-files [hunk 8]
```diff
@@ -1,33 +1,27 @@
 #!/usr/bin/env bash
 
 #
-# This script checks the ownCloud desktop client documentation manual to 
-# see if any of the example files are orphaned.
+# This script checks the ownCloud documentation manual(s) to 
+# see if attachment/example/image files are orphaned.
```
Comment (phil-davis) [inline bin/find-orphaned-example-files:5]:
```suggestion
# see if example files are orphaned.
```

File: bin/find-orphaned-attachment-files [hunk 9]
```diff
@@ -1,33 +1,27 @@
 #!/usr/bin/env bash
 
 #
-# This script checks the ownCloud desktop client documentation manual to 
-# see if any of the example files are orphaned.
+# This script checks the ownCloud documentation manual(s) to 
+# see if attachment/example/image files are orphaned.
```
Comment (phil-davis) [inline bin/find-orphaned-attachment-files:5]:
```suggestion
# see if attachment files are orphaned.
```

File: bin/find-orphaned-image-files [hunk 10]
```diff
@@ -1,33 +1,26 @@
 #!/usr/bin/env bash
 
 #
-# This script checks the ownCloud desktop client documentation manual to 
-# see if any of the example files are orphaned.
+# This script checks the ownCloud documentation manual(s) to 
+# see if attachment/example/image files are orphaned.
```
Comment (phil-davis) [inline bin/find-orphaned-image-files:5]:
```suggestion
# see if image files are orphaned.
```

**Review by phil-davis — APPROVED** [review]
You can choose what review suggestions to take.
I have not attempted to check-and-test every line of bash magic!

File: bin/find-orphaned-base (hunk 4 above)
Comment (mmattel) [reply bin/find-orphaned-base:48]: yes.
It is very helpful to keep all the commented out debugging lines.
I do regular tests, every 3-6 months, and this helps to debug things more quickly.

**Pull request merged**